tests/phpunit/tests/block-supports/duotone.php

```diff
@@ -83,6 +83,29 @@ public function test_get_slug_from_attribute( $data_attr, $expected ) {
 		$this->assertSame( $expected, $reflection->invoke( null, $data_attr ) );
 	}
 
+	/**
+	 * Tests whether the CSS declarations are generated even if the block content is
+	 * empty. We need this to make the CSS output stable across paginations for
```

Comment:
Removing "we" as it's unclear who "we" refers to:

```suggestion
	 * empty. This is needed to make the CSS output stable across paginations for
```